iron-component-page.html

```diff
@@ -609,6 +612,19 @@
           if (!this._findDescriptor(this.active)) {
             this.active = this._getDefaultActive();
           }
+
+          // Scroll to deep link hash.
+          // TODO: support scrolling to hidden API properties/methods.
+          // TODO: deep link into non-default "active" view in dropdown.
+          this.async(function() {
+            var hash = window.location.hash;
+            if (hash) {
+              var goToElement = this.$.view.querySelector(hash);
+              if (goToElement) {
+                goToElement.scrollIntoView();
```

Comment:
oh.  Is this code needed because upgrade happens after the browser automatically triggers this behavior?